experiments/conductor/cmd/runner/mock_commands.go

```diff
@@ -818,7 +821,7 @@ func runMockgcpTests(opts *RunnerOptions, branch Branch) {
 		},
 		WorkDir:    workDir,
 		Env:        map[string]string{"WRITE_GOLDEN_OUTPUT": "1", "E2E_GCP_TARGET": "mock"},
-		MaxRetries: 2,
+        MaxRetries: -1, // no retries, even if user defined
```

Comment:
wont setting this to 1 be sufficient ?

Reply:
we can rename this to maxattempts instead of retries if its more appropriate

Reply:
+1 to @barney-s  and the naming suggestion as well

Reply:
Closing in favor of https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4062